apps/acrs/behaviours/declaration.js

```diff
@@ -0,0 +1,9 @@
+const Utilities = require('../../../lib/utilities');
+
+module.exports = superclass => class extends superclass {
+  locals(req, res) {
+    const locals = super.locals(req, res);
+    locals.isChildrenOver18 = Utilities.isOver18(req.sessionModel.get('date-of-birth'));
+    return locals;
+  }
+};
```

Comment:
I think that the logic required for the conditional rendering of the declaration page is a bit more complex than here - From the Figma it seems like its if they added anyone under 18 to their form (e.g. a child, sibling or additional family member.